cypress/e2e/practise.spec.ts

```diff
@@ -0,0 +1,61 @@
+import homePage from '../support/pages/home.page';
+import fileUpload from '../support/pages/file-upload.page';
+import webInputs from '../support/pages/web-inputs.page';
+import dragNDrop from '../support/pages/drag-n-drop.page';
+import actions from '../fixtures/actions-enums';
+
+describe('Automate different elements', () => {
+  beforeEach(() => {
+    cy.visit(Cypress.config().baseUrl!);
+  });
```

Comment:
We can use before hook to run the setup code only once before all the test cases.